sql/planbuilder/proc.go

```diff
@@ -404,11 +406,55 @@ func (b *Builder) buildBlock(inScope *scope, parserStatements ast.Statements, fu
 			}
 		}
 		stmtScope := b.buildSubquery(inScope, s, ast.String(s), fullQuery)
+		if b.qFlags.IsSet(sql.QFlagCreateProcedure) {
+			b.validateStoredProcedure(stmtScope.node)
+		}
 		statements = append(statements, stmtScope.node)
 	}
+
 	return plan.NewBlock(statements)
 }
 
+func (b *Builder) validateStoredProcedure(node sql.Node) {
+	// For now, we don't support creating any of the following within stored procedures.
+	// These will be removed in the future, but cause issues with the current execution plan.
+	var err error
+	spUnsupportedErr := errors.NewKind("creating %s in stored procedures is currently unsupported " +
+		"and will be added in a future release")
+	transform.Inspect(node, func(n sql.Node) bool {
+		switch n.(type) {
+		case *plan.CreateTable:
```

Comment:
something we could do at some point is aggregate some of these into flags, like for the DDL group